src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Color {
+    red,
+    blue,
+    green,
+    purple,
+    black,
+    white,
+    yellow
```

Comment:
let's write them in upper case
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names